### ISSUE TITLE
Using in a project based on browserify

### ISSUE THREAD

**Issue by lmr2391** [mauricius/vue-draggable-resizable#12]
Hi,
I would like to use vue-draggable-resizable in a project set up using the vue cli with the browserify option. When trying to import VueDraggableResizable I get the error message:

Error: Cannot find module '!!../../node_modules/css-loader/index.js!../../node_modules/vue-loader/lib/style-compiler/index.js?{"id":"data-v-1e95fd0f","scoped":true,"hasInlineConfig":false}!../../node_modules/vue-loader/lib/selector.js?type=styles&index=0!./vue-draggable-resizable.vue' from 'C:\Users\lmrin\tutorials\vue-form2\node_modules\vue-draggable-resizable\dist'

I already tried to install css-loader. However this didn't result in a working import. Does anybody know how to convert this package to be usable in browserify projects? And could you maybe include browserify builds in future versions?

Cheers
Lars

**Comment by mauricius** [comment]
The library works also with Browserify, I just quickly tested it with a the vue-cli browserify-simple template. The error was caused by a couple of hot-module-replacement requires in the build file, even if they were enclosed in a false condition.

I just added (v.1.2.2) the Uglify plugin to the build step, it should minify the resulting file and remove those problematic lines.

Let me know if it works!
